pkg/agent/server/server.go

```diff
@@ -1,179 +0,0 @@
-/*
-Copyright 2019 The Kubernetes Authors.
```

Comment:
I know the deleted "legacy" code was actually legacy. This is also legacy?

Reply:
Yes it is legacy. This is a one-hop proxy server without the dialing back agent.